src/layer/arm/innerproduct_arm.cpp

```diff
@@ -353,7 +354,118 @@ int InnerProduct_arm::forward(const Mat& bottom_blob, Mat& top_blob, const Optio
 
         top_blob[p] = sum;
     }
-
+   }
+    else if(axis==2)
+    {	
```

Comment:
There is not an implementation for packing layout

You have to claim this in create_pipeline()
```
if (axis == 1)
    support_packing = false;
```